opencost/opencost-ui/Makefile

```
@@ -0,0 +1,16 @@
+# OpenCost UI - Build from Source
+# Production builds should use the build-from-source.yaml GitHub workflow
+
+SOURCE_REPO ?= opencost/opencost-ui
+SOURCE_VERSION ?= 1.117.5
```

Can we fail here if the value is not provided instead of having a default value. This value will go stale very soon anyway.

---

Just like `: ${SOURCE_VERSION?"Need to set SOURCE_VERSION"}` nothing fancy.

---

done